my-app/src/index.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import './index.css';
```

Comment:
Consider separating third-party imports from the project's ones.
Also, the css import should be the last one.

Reply:
/fxd